Procfile

```diff
@@ -1,3 +1,2 @@
-release: bundle exec rails db:migrate
-release: bundle exec rails csv:load
+release: release-tasks
```

Comment:
Does this need to be `bin/bash/release-tasks` or does running `release-tasks` from the root of the repo work for you?

Reply:
We can leave it at root. No worries. Let's try it